### PR TITLE
Added QueryMessageFlags client method

### PR DESCRIPTION
**Pull request by gumuz**

### CODE REVIEW

File: message.go [hunk 1]
```diff
@@ -279,25 +279,78 @@ func (c *Client) DeleteMessage(msgID string) error {
...
 	}
 
+	if userID == "" {
+		return errors.New("user ID is empty")
+	}
+
 	options := map[string]interface{}{
 		"target_message_id": msgID,
+		"user_id":           userID,
 	}
```
Comment (gumuz) [inline message.go:354]: these were missing the mandatory userID serverside parameter

File: channel_type_test.go [hunk 2]
```diff
@@ -13,6 +14,8 @@ func prepareChannelType(t *testing.T, c *Client) *ChannelType {
 	ct, err := c.CreateChannelType(ct)
 	require.NoError(t, err, "create channel type")
 
+	time.Sleep(6 * time.Second)
```
Comment (gumuz) [inline channel_type_test.go:17]: I believe there was a latency/timing issue here, where the new ChannelType wasn't directly available to the test that created them

File: .gitignore [hunk 3]
```diff
@@ -14,3 +14,6 @@
 .vscode/
 .idea/
 .envrc
+
+# vendored dependencies and binaries
+vendor/
```
Comment (ferhatelmas) [inline .gitignore:19]: this is actually need since we don't vendor but having doesn't hurt

File: channel_type_test.go (hunk 2 above)
Comment (ferhatelmas) [reply channel_type_test.go:17]: possible

File: message.go (hunk 1 above)
Comment (ferhatelmas) [reply message.go:354]: this is good 👍🏻

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**